docs/installation.md

```diff
@@ -15,13 +14,51 @@ We currently recommend installing the development version of owlkettle.
 $ nimble install owlkettle@#head
 ```
 
+## Properly setting up MSYS2 on Windows
+
+For the Windows OS you'll need the MSYS2 
+that you can install from their [official website](https://www.msys2.org/).
+
+Then, you'll need to use a toolchain with essential binaries as compilers, linkers 
+and the `pkg-config` as well.
+
+There are 2 noteworthy toolchain provided by the MSYS2, choose one of them to install:
```

Comment:
You say

> A) `CLANG64`: the recommended one for Nim
> B) `UCRT64`: the recommended one by default for most projects

this is a bit confusing. When would you prefer A over B or otherwise?

Reply:
You're completely right. I'll change this.

I'll remove the `UCRT` so, since the user will be using Nim. And I'll put `UCRT` and `MINGW` as reference if they want to.

I think that delegate the choice for the user would be a bad idea... Instead of that, I'll stick with the `CLANG` choice.